packages/web-components/src/components/va-privacy-agreement/va-privacy-agreement.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable i18next/no-literal-string */
```

Comment:
Also, can we remove this? Letting the warnings appear helps us know what will need translation.